rotkehlchen/tests/utils/dataimport.py

```diff
@@ -2422,79 +2422,62 @@ def assert_bitcoin_tax_trades_import_results(
         rotki: Rotkehlchen,
         csv_file_name: Literal['bitcoin_tax_trades.csv', 'bitcoin_tax_spending.csv'],
 ):
+    expected_history_events: list[HistoryEvent | SwapEvent]
     if csv_file_name == 'bitcoin_tax_trades.csv':
         expected_history_events = [
-            HistoryEvent(
+            SwapEvent(
                 identifier=1,
                 event_identifier='1xyz',  # just a placeholder as comparison is done without this field  # noqa: E501
```

Comment:
can we maybe assign it to a variable using the walrus operator and reuse? same for the other event. That would avoid repeating the comment

Reply:
i didn't use a walrus operator because another it is used in the elif block.